sysdeps/managarm/generic/fork-exec.cpp

```diff
@@ -148,7 +148,10 @@ int sys_sleep(time_t *secs, long *nanos) {
 	));
 
 	auto element = globalQueue.dequeueSingle();
-	auto result = parseSimple(element);
+	if (!element) {
+		return EINTR;
+	}
+	auto result = parseSimple(*element);
```

Comment:
We have this change for sleep but AFAICT `sleep()` is still not cancelable since we don't integrate it with the cancellation event. We could drop support for `sleep()` in the initial PR.